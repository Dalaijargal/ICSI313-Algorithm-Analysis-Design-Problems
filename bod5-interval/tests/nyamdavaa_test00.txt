10
1 5 5 
3 8 7 
4 9 8 
6 11 1 
1 15 9 
7 14 3 
12 18 4 
13 20 5 
4 4 4 
9 15 6 