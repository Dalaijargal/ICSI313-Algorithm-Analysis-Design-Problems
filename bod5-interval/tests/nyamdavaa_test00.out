13 
0010000100